URIResolver/CMakeLists.txt

```diff
@@ -9,12 +9,15 @@ find_package(Boost REQUIRED)
 find_package(PythonLibs REQUIRED)
 find_package(OpenEXR REQUIRED)
 find_package(TBB REQUIRED)
+find_package(MySQL REQUIRED)
 
 link_directories(${USD_LIBRARY_DIR})
 
-find_library(MYSQL_CONNECTOR_LIBRARY
-             name libmysqlclient.a
-             HINTS ${MYSQL_CONNECTOR_ROOT}/lib)
+find_library(MYSQL_CLIENT_LIBRARY
+		NAMES "libmysqlclient.a"
+		PATHS ${MYSQL_CONNECTOR_ROOT}/lib
+			/usr/lib/x86_64-linux-gnu
```

Comment:
Why do we need to call find_library and find_package at the same time? If there is a find module for MySql, let's use the library found in that module.

Reply:
Sure, will fix.